### ISSUE TITLE
0.2.0: `emojicodes` has install time `setuptools` dependency

### ISSUE THREAD

**Issue by kloczek** [sphinx-contrib/emojicodes#42]
```console
[tkloczko@pers-jacek SPECS]$ grep pkg_ /usr/lib/python3.8/site-packages/sphinxemoji/sphinxemoji.py
from pkg_resources import resource_filename
```
`pkg_resources` module is part of the `setuptools` however this module is marked as deprecated and should be replaced by `importlib-metadata`.
Example of the necessary change https://github.com/dask/distributed/pull/5923

**Comment by Peque** [comment]
Thanks for reporting @kloczek ! I encourage you to open a pull request with the necessary changes. :blush:

**Comment by kloczek** [comment]
Really sorry but I have already busy whole next 7-8 days.
Maybe later ..

**Comment by hotchilipowder** [comment]
Any updates?

**Comment by Peque** [comment]
@hotchilipowder @kloczek Should be fixed in latest release (`0.3.0`).
